app/src/main/res/values/strings.xml

```diff
@@ -96,6 +96,10 @@
     <string name="push_missing_app_info">There are no applications available on the server to push a message to.</string>
     <string name="message_copied_to_clipboard">Content copied to clipboard</string>
     <string name="not_loggedin_share">Cannot share to Gotify, because you aren\'t logged in.</string>
+    <string name="action_dialog_missing">Missing URL</string>
+    <string name="action_dialog_message">You have received a message with a URL to be opened:</string>
```

Comment:
How about we name this more along the existing naming of the feature?
```suggestion
    <string name="action_dialog_message">You have received a message with an intent url:</string>
```

Reply:
Sure